app/models/service/aggregation.rb

```diff
@@ -42,26 +151,36 @@ def aggregate_direct_vm_memory_on_disk
   end
 
   def aggregate_all_vm_cpus
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
+    attributes["aggregate_all_vm_cpus"] || all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
```

Comment:
Minor, but will
```ruby
all_vms.inject(0) { |aggregate, vm| aggregate + vm.cpu_total_cores.to_i }
```
behave the same, avoiding the need to reassign here and elsewhere?